src/screens/ENSSearchSheet.js

```diff
@@ -65,11 +67,11 @@ export default function ENSSearchSheet() {
 
   const handlePressContinue = useCallback(() => {
     Keyboard.dismiss();
-    navigate(Routes.ENS_ASSIGN_RECORDS_SHEET);
-  }, [navigate]);
+    navigate(Routes.ENS_ASSIGN_RECORDS_SHEET, { name: `${searchQuery}.eth` });
```

Comment:
you can remove this, on the main branch, there is an action to set the name in redux